ui/hunter/presets.ts

```diff
@@ -1,27 +1,8 @@
-import { Consumes } from '/tbc/core/proto/common.js';
-import { Drums } from '/tbc/core/proto/common.js';
-import { EquipmentSpec } from '/tbc/core/proto/common.js';
-import { Flask } from '/tbc/core/proto/common.js';
-import { Food } from '/tbc/core/proto/common.js';
-import { ItemSpec } from '/tbc/core/proto/common.js';
-import { Potions } from '/tbc/core/proto/common.js';
-import { WeaponImbue } from '/tbc/core/proto/common.js';
-import { Faction } from '/tbc/core/proto_utils/utils.js';
-import { Player } from '/tbc/core/player.js';
+import * as Tooltips from "/tbc/core/constants/tooltips.js";
+import { Player } from "/tbc/core/player.js";
+import { Consumes, EquipmentSpec, Flask, Food, Potions, WeaponImbue } from "/tbc/core/proto/common.js";
 
-import {
-	Hunter,
-	Hunter_Rotation as HunterRotation,
-	Hunter_Rotation_WeaveType as WeaveType,
-	Hunter_Options as HunterOptions,
-	Hunter_Options_Ammo as Ammo,
-	Hunter_Options_QuiverBonus as QuiverBonus,
-	Hunter_Options_PetType as PetType,
-} from '/tbc/core/proto/hunter.js';
-
-import * as Enchants from '/tbc/core/constants/enchants.js';
-import * as Gems from '/tbc/core/proto_utils/gems.js';
-import * as Tooltips from '/tbc/core/constants/tooltips.js';
+import { Hunter_Options as HunterOptions, Hunter_Options_Ammo as Ammo, Hunter_Options_PetType as PetType, Hunter_Options_QuiverBonus as QuiverBonus, Hunter_Rotation as HunterRotation, Hunter_Rotation_WeaveType as WeaveType } from "/tbc/core/proto/hunter.js";
```

Comment:
Is there a reason for collapsing this onto 1 line? It seems less readable.

Reply:
Ah I must have accidentally checked optimize imports in intellij when I was committing it, that's my bad. I'll fix it.